apps/damap-frontend/src/styles.scss

```diff
@@ -58,6 +58,9 @@
   // mat3 provides colors for error states, but not for success or warning...
   // so it is set to predefined values here
   --success-container: #e9f5d2;
+
+  --success-icon-color: #{mat.get-theme-color($app-light-theme, tertiary, 70)};
```

Comment:
For me the color looks good, @lpandath do you want a different hue?

Reply:
_my opinion_:  As suggested in Discord (the differentiation between fill/outline) about the coloring. However, if you decide to choose an additional color, I would recommend using the lighter shades from the logo itself to ensure it fits well within the app.